packages/lib-classifier/src/plugins/tasks/SubjectGroupComparisonTask/README.md

```diff
@@ -0,0 +1,50 @@
+# Subject Group Comparison Task
+
+This Task allows users to select cells on the Subject Group Viewer's grid.
+Selecting cells is a way for volunteers to say that, "hey, these images look
+different from other images on the grid!"
+
+- This task is STRONGLY ASSOCIATED with the Subject Group Viewer. Please see `src/components/Classifier/components/SubjectViewer/components/SubjectGroupViewer/README.md` for more details.
+- This task is STRONGLY ASSOCIATED with the Subject Group "type" of Subjects. Please see `src/store/SubjectGroup/README.md` for more details.
+
+The Subject Group Comparison Task was originally created by @shaunanoordin on Mar 2020, for the SURVOS project.
+
+## Data Models
+
+Simple Dropdown task data structure, example:
```

Comment:
Wrong task name here?

Reply:
Whoops! Thanks for spotting this, I'll fix it right away.